docs/src/stream.rst

```diff
@@ -224,4 +224,20 @@ API
 
     .. versionchanged:: 1.4.0 UNIX implementation added.
 
+.. c:function:: int uv_enable_largesocket(uv_stream_t* handle)
```

Comment:
done, this is moved to tcp structure, renamed the API to reflect changes being applicable to tcp only, and the document changes moved from stream.rst to tcp.rst.